### DISCUSSION TITLE
Layouts: customizable breakpoints

### DISCUSSION THREAD

**Discussion by markcaron** [RedHat-UX/red-hat-design-system#345] [Ideas]
I'm thinking we should provide CSS custom properties as "hooks" for adjusting Layout breakpoints when needed on specific designs.

For example, what if we want 3 columns on a large screen to break down to 1 column at `900px` rather than our standard `992px` or `768px`?

**Comment by zeroedin** [comment]
I think this related to https://github.com/RedHat-UX/red-hat-design-system/discussions/278
